trl/trainer/callbacks.py

```diff
@@ -273,6 +278,30 @@ def on_train_begin(self, args: TrainingArguments, state: TrainerState, control:
                 generation_config=self.generation_config,
                 batch_size=args.per_device_eval_batch_size,
             )
+            # Compute initial win rate as a reference point
```

Comment:
I've added this at the start of training because it is very helpful for debugging to see if the model generations are sensible or not

Reply:
Do we need to get the winner indices here? We just compare the ref against itself.